tests/func/test_get.py

```diff
@@ -87,3 +89,43 @@ def test_get_to_dir(dname, erepo):
 
     assert os.path.isdir(dname)
     assert filecmp.cmp(erepo.FOO, dst, shallow=False)
+
+
+def test_get_from_non_dvc_master(empty_dir, git_erepo, caplog):
+    storage = git_erepo.mkdtemp()
+
+    dvc_branch = "dvc_test"
+    git_erepo.git.git.checkout("master", b=dvc_branch)
+
+    dvc_repo = Repo.init(git_erepo._root_dir)
```

Comment:
So you make a dvc repo in master branch, looks like this breaks the purpose of the test.

Reply:
As far as I know line 98 is equivalent to:
`git checkout -b dvc_test`